tools/topology/topology1/sof-acp-rmb-tdm8ch-dmic2ch-nau8825-max98360.m4

```diff
@@ -1,5 +1,5 @@
 #
```

Comment:
minor nit: Given this only changes "sof-acp*" files, a prefix like "topology1: acp: " would be nice.

Reply:
used topology1:amd